features/cellular/framework/AT/AT_CellularNetwork.h

```diff
@@ -106,6 +106,11 @@ class AT_CellularNetwork : public CellularNetwork, public AT_CellularBase {
      */
     virtual nsapi_error_t set_access_technology_impl(RadioAccessTechnology op_rat);
 
+    /** Reduce copy-paste by starting context looping with different command but
+     *  same response. Can be overridden by the target class.
+     *
+     */
+    virtual void set_context_command();
```

Comment:
would get_context_state_command() be better name and comment:
"... Sends command to query the active state of the PDP contexts... Can be overridden by the target class."

Reply:
@jarvte ^^^

Reply:
fixed as suggested